docs/user_manual.md

```diff
@@ -272,7 +273,10 @@ For example, `{"foo": {"internalPort": 4711}}`  will  map the internal port
 4711 of the container to an arbitrary external port on the host. 
   
 Specifying `{"foo": {"internalPort": 4711, "externalPort": 80}}` will map
-internal port 4711  of the container to port 80 on the host. 
+internal port 4711  of the container to port 80 on the host.
+
+The `ip` attribute is optional and specifies the IP address on which the port
+will be exposed. Any IP string literal is valid.
```

Comment:
And in other words this is likely only useful if you want to deploy a job that exposes a port mapping that only listens on localhost